packages/admin/cms-admin/src/documents/types.ts

```diff
@@ -47,4 +47,5 @@ export interface DocumentInterface<
     anchors: (input: DocumentInput) => string[];
     dependencies: (input: DocumentInput) => BlockDependency[];
     replaceDependenciesInOutput: (output: DocumentOutput, replacements: ReplaceDependencyObject[]) => DocumentOutput;
+    hasNoPreview?: boolean;
```

Comment:
This type could be changed to either `true` or `false` to only allow the intended use case.